web/templates/ticket.html

```diff
@@ -20,98 +20,166 @@
 
   {{- $tick := . -}}
 
-    <div class="top">
-      <div id="topbar" class="box">
-        Ticket #{{ .Id }}: {{ .Subject }}
-      </div>
+    <main role="main">
 
-      <div class="box" id="basics">
-        <h2>Basics</h2>
-        <div class="row">Id: {{ .Id }}</div>
-        <div class="row">Status: {{ .Status }}</div>
-        {{/* Need to get name for .Queue id: <div class="row">Queue: {{ .Queue }}</div> */}}
-      </div>
+      <!-- Main jumbotron for a primary marketing message or call to action -->
 
-      <div class="box" id="dates">
-        <h2>Dates</h2>
-        <div class="row">Created: {{ .Created }}</div>
-        <div class="row">Last Updated: {{ .LastUpdated }}
-          by {{ obfuscateEmail .LastUpdatedBy.RealName }} </div>
-        <div class="row">Closed: {{ .Closed }}</div>
+      <div class="jumbotron">
+        <div class="container">
+          <h2>Reading Perl 5 Static RT #{{ .Id }}</h2>
```

Comment:
The "Reading..." part...

Reply:
are you complaining about the `height` or the content? not sure what action is required there, thanks for the clarification

Reply:
The content.  In the h2, it's a little weird.